src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,17 +3,88 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int MAX_VALUE_ARRAY = 10;
+    private K key;
+    private V value;
+    private Object[] items = new Object[MAX_VALUE_ARRAY];
```

Comment:
```suggestion
    private StorageNode[] items;
```
move initialization to empty constructor